### PR TITLE
updated Logos to SVGs and updated the favicon

### PR DESCRIPTION
**Pull request by thatbreezeofmine**

### CODE REVIEW

File: apps/app/src/components/templates/layout.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const Layout = () => {
       {isBrowser ? (
         <div className="flex w-screen h-screen bg-current">
           <div className="flex m-auto flex-col">
-            <img src="/logo-light.png" alt="logo" className="h-32 mx-auto my-8" />
+            <img src="/logo-light.svg" alt="logo" className="h-32 mx-auto my-8" />
```
Comment (Stormix) [inline apps/app/src/components/templates/layout.tsx:22]: Can you make this change depending on light theme or dark theme ?

**Review by Stormix — CHANGES_REQUESTED** [review]
One small change

File: apps/app/src/components/templates/layout.tsx (hunk 1 above)
Comment (Stormix) [reply apps/app/src/components/templates/layout.tsx:22]: useTheme() should be available here

Comment (Stormix) [reply apps/app/src/components/templates/layout.tsx:22]: (Providers has the theme provider)

Comment (thatbreezeofmine) [reply apps/app/src/components/templates/layout.tsx:22]: sure

Comment (thatbreezeofmine) [reply apps/app/src/components/templates/layout.tsx]: Done

**Review by Stormix — APPROVED** [review]

**Pull request merged**